components/compositing/compositor.rs

```diff
@@ -1565,6 +1575,25 @@ impl IOCompositor {
                     },
                     _ => {},
                 }
+
+                match self.lcp_detector.calculate_largest_contentful_paint(
+                    paint_time,
+                    current_epoch,
+                    pipeline_id.into(),
+                ) {
+                    None => {},
+                    Some(lcp) => {
+                        #[cfg(feature = "tracing")]
+                        let _ = tracing::debug_span!(
+                            "largest-contentful-paint",
+                            servo_profiling = true,
+                            timestamp = (lcp.paint_time - CrossProcessInstant::epoch())
+                                .whole_microseconds(),
+                        )
+                        .entered();
+                        println!("largest_contentful_paint: {:?}", lcp);
+                    },
+                }
```

Comment:
This should be `if let Some(...) =` and it also looks like the `println!` has been left over rom testing.